src/physics/cam/yog_intr.F90

```diff
@@ -226,6 +232,34 @@ subroutine yog_tend(ztodt, state, ptend)
    call outfld('YOGDICE ',ptend%q(1,1,ixcldice) ,pcols   ,lchnk   )
    call outfld('YOGDLIQ ',ptend%q(1,1,ixcldliq) ,pcols   ,lchnk   )
    call outfld('YOGPREC ',yog_precsfc ,pcols   ,lchnk   )
+
+   ! Update the number concentration tendencies for liquid and ice species
+   ! Values taken to match those in the `clubb_tend_cam()` subroutine
+   ! YOG can produce negative number tendency, but we must ensure it doesn't reduce total
+   ! number concentration below 0.0
+   call cnst_get_ind('NUMLIQ', ixnumliq)
+   call cnst_get_ind('NUMICE', ixnumice)
+   do k = 1, pver
+   do i = 1, ncol
```

Comment:
Minor comment and probably personal taste, but I would indent the inner loop, for clarity

Reply:
I like writing it in this form to indicate that all code is enclosed inside the loops indicating a loop over each atmospheric cell, with no special treatment for grid columns outside of this.